src/froll.c

```diff
@@ -49,7 +45,7 @@ void frollmean(unsigned int algo, double *x, uint64_t nx, ans_t *ans, int k, int
  */
 void frollmeanFast(double *x, uint64_t nx, ans_t *ans, int k, double fill, bool narm, int hasna, bool verbose) {
   if (verbose)
-    snprintf(end(ans->message[0]), 500, "%s: running for input length %"PRIu64", window %d, hasna %d, narm %d\n", __func__, (uint64_t)nx, k, hasna, (int)narm);
+    snprintf(end(ans->message[0]), 500, "frollmeanFast: running for input length %"PRIu64", window %d, hasna %d, narm %d\n", (uint64_t)nx, k, hasna, (int)narm);
```

Comment:
For translation purposes, it will be simpler to replace `__func__` with `"frollmeanFast"`, then only the format message has to be translated, rather than translating essentially the same things a bunch of times.

Took the idea from `base` which does this often.

I'll update this in the `translation` branch shortly

Reply:
Ok. But does that pass the new mingw-w64 compiler in rtools4 (win-builder ATC)?  The potential compiler glitch might pertain to the `%s` and be unrelated to the `__func__`.  Which is why I went straight for not using %s at all first to save a step to see if that passed, and it did to my surprise.  Before putting back the %s we have to check whether the problem is %s or __func__.

Reply:
makes sense. is there such a test in our GL CI pipeline?

Reply:
not sure how hard that is to setup, @jangorecki? Maybe it's just a case of pointing to (new) rtools40, or could be more involved.

Reply:
Update:  https://github.com/Rdatatable/data.table/pull/3935#discussion_r362137293

Reply:
@mattdowle I missed your previous comment. Good we know that `__func__` was causing problem when used with int64 printing. If rtools40 would help about this, to we want to use `__func__` together with int64 printing? It is just a compilation warning so not really an issue.

Reply:
@jangorecki by 'if rtools40 would help about this' do you mean if we report to mingw-w64 team, they accept it as bug, they fix the bug, and then Rtools4.0 upgrades to the fixed version of gcc, do we then want to put the `__func__` back?  If so, I guess yes, because the code in this file is currently confusing because it uses `__func__` everywhere except those 4 calls that include the int64 too. It's strange to have that difference so it has to be commented to explain, and I suppose we need something in CRAN_Release.cmd to catch that from coming back in future. Is GLCI using Rtools4.0 yet and does GLCI search the install log on windows to catch compiler warnings like this one?

Reply:
we use 3.5.0.4 on gitlab builds, will upgrade to recent stable